AnimationEncoder/doxy.cfg

```diff
@@ -1560,6 +1702,190 @@ EXTRA_SEARCH_MAPPINGS  =
 
 GENERATE_LATEX         = NO
 
+# The LATEX_OUTPUT tag is used to specify where the LaTeX docs will be put. If a
```

Comment:
I deleted the next ~180 lines manually since line 1703 says `NO`. Similarly for all the other backends that we don't use.

I don't mind much though.